doctr/models/detection/differentiable_binarization/pytorch.py

```diff
@@ -39,6 +39,15 @@
         'std': (0.264, 0.2749, 0.287),
         'url': 'https://github.com/mindee/doctr/releases/download/v0.3.1/db_mobilenet_v3_large-fd62154b.pt',
     },
+    'db_resnet50_rotation': {
+        'backbone': resnet50,
+        'backbone_submodule': None,
+        'fpn_layers': ['layer1', 'layer2', 'layer3', 'layer4'],
```

Comment:
You can remove all those 3 dict entries (cf. the refactoring and the other dict entries)